patcher/src/widgets/CharacterCreation/services/session/races.ts

```diff
@@ -40,10 +40,10 @@ export function fetchRacesSuccess(races: RaceInfo[]) {
   };
 }
 
-export function fetchRacesFailed(error: ResponseError) {
+export function fetchRacesFailed(error: any) {
```

Comment:
`ResponseError` is defined by patcher character creation widget, whereas value.error is defined by the GameDataAPI.  What we really need is for GameDataAPI to define some types.